src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrderPayeeInfoDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SwedbankPay.Sdk.Common;
+using System;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrderPayeeInfoDto
+    {
+        public Uri Id { get; set; }
+        public string PayeeId { get; set; }
+        public string PayeeReference { get; set; }
+        public string PayeeName { get; set; }
+        public string CorporationId { get; set; }
+        public string CorporationName { get; set; }
+        public string ProductCategory { get; set; }
+
+        internal PayeeInfo Map()
+        {
+            if (string.IsNullOrEmpty(PayeeId))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)